### PR TITLE
code moved towards pep_8 conventions for tax_smoothing_2

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

File: source/rst/tax_smoothing_2.rst [hunk 1]
```diff
@@ -406,12 +407,12 @@ moving from one to the other.
                   [0.1, 0.9]])
 
     # Construct and solve the model using the LQMarkov class
-    MJLQBarro = qe.LQMarkov(Π, Qs, Rs, As, Bs, Cs=Cs, Ns=Ws, beta=β)
-    MJLQBarro.stationary_values()
+    mjlq_barro = qe.LQMarkov(Π, Qs, Rs, As, Bs, Cs=Cs, Ns=Ws, beta=β)
```
Comment (AnjuJoon) [inline source/rst/tax_smoothing_2.rst]: @jstac does this looks fine?
this is an instance of the class LQMarkov

Comment (jstac) [reply source/rst/tax_smoothing_2.rst]: Thanks @AnjuJoon 

I suggest something simpler.  How about just `lqm`?  Then later you'll have `lqm2`, etc.

Comment (AnjuJoon) [reply source/rst/tax_smoothing_2.rst]: yes, that's a good idea @jstac lqm in place of MJLQBarro?

Comment (jstac) [reply source/rst/tax_smoothing_2.rst]: Yes please

**Comment by AnjuJoon** [comment]
@jstac this is ready to merge

**Pull request merged**